pkg/processor/runtime/java/docker/src/main/java/io/nuclio/processor/Wrapper.java

```diff
@@ -149,14 +149,18 @@ public static void main(String[] args) throws Throwable {
 
         Context context = new WrapperContext(sock.getOutputStream());
         Response response;
+        Long start, end;
 
         while (true) {
             try {
                 Event event = eventReader.next();
                 if (event == null) {
                     break;
                 }
+                start = System.currentTimeMillis();
                 response = handler.handleEvent(context, event);
+                end = System.currentTimeMillis();
+                responseEncoder.encodeMetrics(end - start);
```

Comment:
Hey @onesolpark thank you for your contribution,

I would use `encodeMetrics` within the `finally` block to ensure failed events to be monitored as well

Reply:
That was kinda of my initial thought at first and I kinda of got worried about initializing the variables. 
But I think it makes more sense. Please check the changes I made.

Reply:
No problem. Thank you for taking care of it.